### PR TITLE
Fix variables that seem to not be modified to consts.

### PR DESCRIPTION
**Pull request by shahzadlone**
Based on this when I pointed out this to be const but it was too late, it was merging soon: https://github.com/tensorflow/tensorflow/pull/25269#discussion_r252492749

Please guide me incase I can make this even better.

### CODE REVIEW

File: tensorflow/stream_executor/cuda/cuda_dnn.cc [hunk 1]
```diff
@@ -3675,7 +3675,7 @@ bool CudnnSupport::DoTransformTensor(Stream* stream,
   CudnnTensorDescriptor output_tensor_desc(
       output_desc, ToCudnnDataType(output_type, output_desc.layout()));
   auto cudnn = cudnn_->GetHandle(parent_, stream);
-  auto status = [&] {
+  auto const status = [&] {
```
Comment (duncanriach-nvidia) [inline tensorflow/stream_executor/cuda/cuda_dnn.cc]: `const auto`, to match the others?

Comment (shahzadlone) [reply tensorflow/stream_executor/cuda/cuda_dnn.cc]: resolved!

**Review by drpngx — APPROVED** [review]
Yay for const!

**Comment by shahzadlone** [comment]
> Yay for const!

Thanks =P, Glad someone else is also as happy as me to see ```const```.

**Pull request merged**